doc/deployment/deploy_custom.md

```diff
@@ -84,203 +43,219 @@ pachctl deploy custom --persistent-disk <persistent disk backend> --object-store
+You can use the `--local-roles` flag to change the kind of role the `pachyderm` service account uses 
+from cluster-wide (`ClusterRole`) to namespace-specific (`Role`). 
+Using `--local-roles` inhibits your ability to use the [coefficient parallelism](http://docs.pachyderm.io/en/latest/reference/pipeline_spec.html#parallelism-spec-optional) feature.
+After you set the `--local-roles` flag, 
+you might see the following message in the `pachd` pod Kubernetes logs:
 
 ```
 ERROR unable to access kubernetes nodeslist, Pachyderm will continue to work but it will not be possible to use COEFFICIENT parallelism. error: nodes is forbidden: User "system:serviceaccount:pachyderm-test-1:pachyderm" cannot list nodes at the cluster scope
 ```
 
 #### Resource requests and limits
 
-Larger deployments may need to request more resources for `pachd` and `etcd`
+Larger deployments might require you to configure more resources for `pachd` and `etcd`
 or set higher limits for transient workloads.
-They are used to set attributes which are passed on to Kubernetes directly via the produced manifest.
-None of these flags should be modifed from the default values for production deployments without consulting Pachyderm support.
-- `--etcd-cpu-request`: the number of cores Kubernetes should give `etcd`. Fractions are allowed. 
-- `--etcd-memory-request`: the amount of memory Kubernetes should give `etcd`. Accepts the SI suffixes that Kubernetes accepts for such values.
-- `--no-guaranteed`: turn of QoS for `etcd` and `pachd`. Not to be used in production environments.
-- `--pachd-cpu-request`: the number of cores Kubernetes should give `pachd`. Fractions are allowed. 
-- `--pachd-memory-request`: the amount of memory Kubernetes should give `pachd`. Accepts the SI suffixes that Kubernetes accepts for such values.
-- `shards`: the maximum number of `pachd` nodes allowed in the cluster. Increasing this number from the default value of 16 may result in degraded performance.
+The following flags set attributes which are passed on to Kubernetes directly through the produced manifest.
+Do not modify the default values of these flags for production deployments without consulting with Pachyderm support.
+- `--etcd-cpu-request`: The number of CPU cores that Kubernetes allocates to `etcd`. Fractions are allowed. 
+- `--etcd-memory-request`: The amount of memory that Kubernetes allocates to `etcd`. The SI suffixes are accepted as possible values.
+- `--no-guaranteed`: Turn off QoS for `etcd` and `pachd`. Do not use this flag in production environments.
+- `--pachd-cpu-request`: The number of CPU cores that Kubernetes allocates to `pachd`. Fractions are allowed. 
+- `--pachd-memory-request`: The amount of memory that Kubernetes allocates to `pachd`. This flag accepts the SI suffixes.
+- `shards`: The maximum number of `pachd` nodes allowed in the cluster. Increasing this number from the default value of 16 might result in degraded performance.
```

Comment:
Can you explain what this actually does to me? Is it literally just a weird limit on the total number of pachd nodes? that doesn't really make sense.